src/main/java/org/entando/kubernetes/service/digitalexchange/job/EntandoBundleInstallService.java

```diff
@@ -187,11 +187,15 @@ private EntandoBundleJobEntity createInstallJob(EntandoDeBundle bundle, EntandoD
         job.setProgress(0);
         job.setStatus(JobStatus.INSTALL_CREATED);
 
-        try {
-            job.setInstallPlan(null != installPlan ? objectMapper.writeValueAsString(installPlan) : null);
-        } catch (JsonProcessingException e) {
-            log.error("Error converting the received install plan to string", e);
-            job.setInstallPlan(null);
+        if (installPlan != null) {
+            job.setCustomInstallation(installPlan.isCustomInstallation());
+
+            try {
+                job.setInstallPlan(objectMapper.writeValueAsString(installPlan));
+            } catch (JsonProcessingException e) {
+                log.error("Error converting the received install plan to string", e);
```

Comment:
@firegloves not sure about this, a non-json install plan is a strange condition that should lead to a http status 422.
Are you may be checking the format before this code?

Reply:
I added it because `objectMapper.writeValueAsString` declares to throw the `JsonProcessingException`. I could move it elsewhere but the serialization implies the exception management